bindings/kotlin/ldk-node-android/README.md

```diff
@@ -31,7 +31,7 @@ Publishing new version guide.
     - add dependency in `libs.versions.toml`:
         ```toml
         # by tag
-        ldk-node-android = { module = "com.github.ovitrif:ldk-node", version = "v0.6.0-rc.1"
+        ldk-node-android = { module = "com.github.ovitrif:ldk-node", version = "v0.6.0-rc.4"
```

Comment:
`com.github.ovitrif:ldk-node` can be updated to `com.github.synonymdev:ldk-node` please 🙏🏻 , tested and it works great

on 2 lines: `34` and `37`

Reply:
Updated [here](https://github.com/synonymdev/ldk-node/pull/8/commits/36dd8136735ab1675d070d26db08e3f0e1ee9bc4).